contracts/Minter.sol

```diff
@@ -24,21 +24,26 @@ contract SPSMinter {
   uint256 constant public poolsCap = 100;
   /// @notice Maximum amount per block to each pool
   uint256 constant public maxToPoolPerBlock = 50 ether;
+  // @notice minimum payout before it's rounded to 0
+  uint256 constant public minimumPayout = 0.1 ether;
 
   /// @notice Struct to store information about each pool
   struct Pool {
     address receiver;
     uint256 amountPerBlock;
+    uint256 reductionBlocks;
+    uint256 reductionBps;
+    uint256 lastUpdate;
   }
   /// @notice Array to store all pools
   Pool[] public pools;
 
   /// @notice Emitted when mint() is called
   event Mint(address indexed receiver, uint256 amount);
   /// @notice Emitted when pool is added
-  event PoolAdded(address indexed newReceiver, uint256 newAmount);
+  event PoolAdded(address indexed newReceiver, uint256 newAmount, uint256 newReductionBlocks, uint256 newReductionPercentage, uint256 newLastUpdate);
```

Comment:
should says `newReductionBps` instead of `newReductionPercentage`

Reply:
Fixed: https://github.com/steem-monsters/sps-minter/commit/4b55a65d9191dd2d10a88eafa6a33d7031534408